roles/openshift_master_certificates/tasks/main.yml

```diff
@@ -20,6 +20,8 @@
     - admin.kubeconfig
     - master.kubelet-client.crt
     - master.kubelet-client.key
+    - "{{ 'master.proxy-client.crt' if openshift.master.include_proxy_client_cert else omit }}"
+    - "{{ 'master.proxy-client.key' if openshift.master.include_proxy_client_cert else omit }}"
```

Comment:
Have you tested this? I wouldn't expect this to be available unless you set it through openshift_facts and not just setting it with set_fact in the master role.